hawkular-accounts-feature-pack/src/main/resources/content/standalone/configuration/hawkular-realm.json

```diff
@@ -0,0 +1,115 @@
+{
+  "id" : "hawkular-realm",
+  "realm" : "hawkular",
+  "enabled" : true,
+  "sslRequired" : "none",
+  "passwordCredentialGrantAllowed" : true,
+  "registrationAllowed" : true,
+  "requiredCredentials" : [ "password" ],
+  "passwordPolicy" : "length(8) and lowerCase(1) and upperCase(1) and specialChars(1) and digits(1) and hashIterations(1)",
+  "loginTheme" : "hawkular",
+  "accountTheme" : "hawkular",
+  "adminTheme" : "hawkular",
+  "defaultRoles" : [ "user" ],
+  "eventsListeners" : [ "hawkular-jms", "jboss-logging" ],
+  "users" : [ {
+    "id" : "28026b36-8fe4-4332-84c8-524e173a68bf",
+    "username" : "jdoe",
+    "enabled" : true,
+    "totp" : false,
+    "emailVerified" : false,
+    "firstName" : "John",
+    "lastName" : "Doe",
+    "email" : "jdoe@acme.com",
+    "credentials" : [ {
+      "type" : "password",
+      "hashedSaltedValue" : "ltSl89y75f8LTzoT646xO9NPTrm114ubTauplgw+TQ62ZDy8K89+y7jFzp/jnSvEh9TENQ4RRvNA6vdGOUDcXg==",
+      "salt" : "m8PY6Q9lICl9BHSS03D2Ug==",
+      "hashIterations" : 1,
+      "temporary" : false
+    } ],
+    "requiredActions" : [ ],
+    "realmRoles" : [ "user", "offline_access" ],
+    "applicationRoles" : {
+      "account" : [ "view-profile", "manage-account" ]
+    }
+  } ],
+  "roles" : {
+    "realm" : [
+      {
+        "name": "user",
+        "description": "User privileges",
+        "composite": true,
+        "composites": {
+          "application": {
+            "hawkular-accounts-backend": ["user"]
+          }
+        }
+      },
+      {
+        "name": "admin",
+        "description": "Admin privileges",
+        "composite": true,
+        "composites": {
+          "application": {
+            "hawkular-accounts-backend": ["admin"]
+          },
+          "realm": ["user"]
+        }
+      }
+    ],
+    "application": {
+      "hawkular-accounts-backend": [
+        {
+          "name": "user"
+        },
+        {
+          "name": "admin"
+        }
+      ]
+    }
+  },
+  "scopeMappings": [
+    {
+      "client": "hawkular-ui",
+      "roles": ["user"]
+    }
+  ],
+  "applicationScopeMappings" : {
+    "account" : [ {
+      "client" : "hawkular-ui",
+      "roles" : [ "manage-account", "view-profile" ]
+    } ]
+  },
+  "applications" : [
+    {
+      "name": "hawkular-accounts-backend",
+      "enabled": true,
+      "bearerOnly" : false,
+      "publicClient": false,
+      "redirectUris": ["/*", "http://localhost:2772/*"],
+      "webOrigins": ["http://localhost:2772"],
+      "secret" : "${uuid.hawkular.accounts.backend}"
+    },
+    {
+      "name": "hawkular-ui",
+      "enabled": true,
+      "bearerOnly": false,
+      "publicClient": true,
+      "secret": "${uuid.hawkular.ui}",
```

Comment:
Thanks for pointing this out. This is probably the reason my test distro is not passing.